### PR TITLE
Use unique DCOS_DIR for every windows build

### PR DESCRIPTION
**Pull request by janisz**
Fixes: https://jira.d2iq.com/browse/D2IQ-72652
Refs: https://github.com/dcos/dcos-core-cli/pull/400

### CODE REVIEW

File: ci/integration-tests-windows.groovy [hunk 1]
```diff
@@ -63,13 +63,15 @@ pipeline {
...
-                withEnv(["DCOS_TEST_URL=${master_ip}", "OS=${os}"]) {
+                withEnv(["DCOS_TEST_URL=${master_ip}", "OS=${os}", "BUILD_NUMBER=${BUILD_NUMBER}"]) {
                     withCredentials(credentials) {
                         bat '''
                             bash -exc " \
                             export PYTHONIOENCODING=utf-8; \
                             export CLI_TEST_SSH_USER=centos; \
                             export CLI_TEST_MASTER_PROXY=1; \
+                            export DCOS_DIR=$PWD/$BUILD_NUMBER; \
+                            mkdir -p $DCOS_DIR; \
```
Comment (pierrebeitz) [inline ci/integration-tests-windows.groovy]: dcos-cli does this on its own, doesn't it? at least on dcos-ui-CI we don't need to do this i believe.

**Review by pierrebeitz — APPROVED** [review]

File: ci/integration-tests-windows.groovy (hunk 1 above)
Comment (janisz) [reply ci/integration-tests-windows.groovy]: yes by default it creates `.dcos` under `$HOME` here our homedir was not cleaned so the previous build interfere with the new one. 
I manually tested it with Jenkins https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/mesosphere-dcos-cli/job/core/job/integration-tests-windows/1968/console

**Comment by janisz** [comment]
@mesosphere-ci run linux integration tests

**Comment by janisz** [comment]
This change was merged in https://github.com/dcos/dcos-core-cli/pull/517

**Pull request closed without merging**